### PR TITLE
feat(sdk): add more information about current version

### PR DESCRIPTION
**Pull request by yesnault**
this commit will fix the prerequesite plugin by choosing the right plugin arch

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report it#6143.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report ut-engine#6143.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report it#6147.0 ✘  
* w-it
  * w-it ✘

File: engine/api/plugin_test.go [hunk 1]
```diff
@@ -7,13 +7,12 @@ import (
...
 	"os"
 	"path"
-	"runtime"
 	"testing"
 
 	"github.com/ovh/cds/engine/api/test"
 )
 
-const dummyBinaryFile = "https://github.com/ovh/cds/releases/download/0.8.1/plugin-download-" + runtime.GOOS + "-amd64"
+//const dummyBinaryFile = "https://github.com/ovh/cds/releases/download/0.8.1/plugin-download-" + sdk.GOOS + "-amd64"
```
Comment (sguiheux) [inline engine/api/plugin_test.go]: remove?

**Pull request merged**